content/en/opentelemetry/interoperability/instrumentation_libraries.md

```diff
@@ -13,7 +13,12 @@ algolia:
   tags: ['otel instrumentation']
 ---
 
-Datadog supports OpenTelemetry-compatible instrumentations which provides observability for libraries not covered by Datadog SDKs without changing SDKs. You can extend Datadog's tracing capabilities to these frameworks.
+[Instrumentation][1] is the process of adding code to your application to capture and report observability data.
+[Automatic instrumentation][2] is a way to instrument applications and libraries without modifying their source code.
+Both OpenTelemetry and Datadog provide automatic instrumentation in their SDKs.
+
+Datadog SDKs support adding [OpenTelemetry instrumentation libraries][3] to their existing automatic instrumentation.
+This provides observability for libraries not covered by Datadog SDKs without changing SDKs.
```

Comment:
This looks like you may have been working off a previous commit. Is this intentional or can it be reverted?